### PR TITLE
Update README with relevant information

### PR DESCRIPTION
**Pull request by jjbrosnan**
Update README.md with installation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+- **[`metriccentury`](https://github.com/mikeblas/samples-junk/tree/main/metriccentury)** - Data recorded from a 100 km bike ride
 
+## Description
+
+Each folder in this repository has two items within:
+
+ - `README` - An explanation of everything about the data
+ - `csv` - A folder with all relevant CSV files
+
+All data in this repository is stored in standard CSV format.  The `README` in each directory describes everything you need to know about the data.
```
Comment (chipkent) [inline README.md:18]:
```suggestion
```
Seems repetitive.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+## Installation Instructions
+
+1. Follow the README instructions on [Deephaven Community Core](https://github.com/deephaven/deephaven-core) for installing the OSS client and all required dependencies.
+2. From the root of your `deephaven-core` clone, run:
+   1. `docker build -t deephaven/examples samples`
+   2. `docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples download`
+
+## After Installation
+
+The commands above populate `docker/core/data/examples` in the `deephaven-core` clone with all of the data.
```
Comment (chipkent) [inline README.md:29]: Should have a note about stuff getting mounted to `/data` in the docker image.

**Review by chipkent — CHANGES_REQUESTED** [review]

**Review by margaretkennedy — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+## Installation Instructions
+
+1. Follow the README instructions on [Deephaven Community Core](https://github.com/deephaven/deephaven-core) for installing the OSS client and all required dependencies.
+2. From the root of your `deephaven-core` clone, run:
+   1. `docker build -t deephaven/examples samples`
+   2. `docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples download`
+
+## After Installation
+
+The commands above mount `docker/core/data/examples` in the `deephaven-core` clone with all of the data.  This corresponds to `/data` within the docker container.
```
Comment (chipkent) [inline README.md:30]:
```suggestion
The commands above mount `docker/core/data/examples` in the `deephaven-core` clone as `/data` within the docker container.
```

File: gsod/README.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ There are fourteen fields recorded in the observation file. The `USAF` and `WBAN
 - **MaxGustSpeedMPH:** peak guest wind speed in miles per hour
 - **MaxTemperatureF:** maximum temperature, in Fahrenheit
 - **MinTemperatureF:** minimum temperature, in Fahrenheit
-- **PreciptitationInches:** total precipitation, in inches
+- **PrecipitationInches:** total precipitation, in inches
```
Comment (chipkent) [inline gsod/README.md:45]: This data is generated from scripts in `examples-private`.  Review those scripts for the spelling problem.

**Review by chipkent — CHANGES_REQUESTED** [review]

File: gsod/README.md (hunk 4 above)
Comment (jjbrosnan) [reply gsod/README.md:45]: Created a PR in that repo with Mike B as a reviewer to fix the spelling error

**Review by chipkent — APPROVED** [review]

**Pull request merged**